### PR TITLE
Fix Ukrainian localization

### PR DESCRIPTION
**Pull request by zhovner**

### CODE REVIEW

**Comment by mattock** [comment]
Any reviewers for this one? If not, we may need to take the "merge and fix later as necessary" approach.

**Comment by ValdikSS** [comment]
I asked guy who contributed Ukrainian translation before, hope he'll check it soon.

File: res/openvpn-gui-res-ua.rc [hunk 1]
```diff
@@ -183,17 +184,19 @@ FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_UKRAINIAN, SUBLANG_DEFAULT
 BEGIN
     ICON ID_ICO_APP, 0, 8, 16, 21, 20
-    LTEXT "OpenVPN GUI v" PACKAGE_VERSION " - графічний інтерфейс Windows для OpenVPN\n" \
+    LTEXT "OpenVPN GUI v" PACKAGE_VERSION " - графічний інтерфейс OpenVPN для Windows\n" \
           "Copyright (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n" \
-          "http://openvpn.se/", 0, 36, 15, 206, 26
+          "Copyright (C) 2008-2014 Heiko Hund <heikoh@users.sf.net>\n" \
+          "Copyright (C) 2012-2016 OpenVPN GUI contributors\n" \
+          "https://github.com/OpenVPN/openvpn-gui/", 0, 36, 15, 206, 42
```
Comment (selvanair) [inline res/openvpn-gui-res-ua.rc]: Need more vertical space. Change 42 to 50

File: res/openvpn-gui-res-ua.rc [hunk 2]
```diff
@@ -183,17 +184,19 @@ FONT 8, "Microsoft Sans Serif"
...
-	  "і з компрессією пакетів.\n" \
+          "через єдиний UDP або TCP-порт із підтримкой підтверження подлінності сесій " \
+      "і пересилкой(обміном) ключів SSL/TLS, шифруванням, аутентіфікацієй " \
+      "і з компрессією пакетів.\n" \
           "\n" \
-          "Copyright (C) 2002-2005 OpenVPN Solutions LLC <info@openvpn.net>\n" \
+          "Copyright (C) 2002-2016 OpenVPN Technologies, Inc <info@openvpn.net>\n" \
           "With support from - За підтримки - При подддержке http://picku.pp.ua/\n" \
-          "http://openvpn.net/", 0, 8, 45, 235, 56
+          "https://openvpn.net/", 0, 8, 61, 240, 64
```
Comment (selvanair) [inline res/openvpn-gui-res-ua.rc]: And this has to move down: change 61 to 70

**Comment by ValdikSS** [comment]
The person who contributed Ukrainian translation before approved this translation

**Comment by mattock** [comment]
@ValdikSS : ok, merging.

**Comment by ValdikSS** [comment]
@mattock no, wait a min

**Comment by ValdikSS** [comment]
@mattock damn, I'm late. All right then, nevermind.

**Comment by selvanair** [comment]
Some text in the about page is clipped (see my previous comments). I'll fix it in PR 100.

**Pull request merged**